internal/pkg/pluginengine/check.go

```diff
@@ -0,0 +1,30 @@
+package pluginengine
+
+import (
+	"log"
+
+	"github.com/merico-dev/stream/internal/pkg/configloader"
+)
+
+// CheckHealthy return true while all tools are healthy
```

Comment:
CheckHealthy return true while all tools are healthy ->

CheckHealthy return**s** true if all tools are healthy

Reply:
get it